### PR TITLE
Apply fixing lsp-clients-angular-language-server-command.

### PR DESCRIPTION
**Pull request by XenHunt**
This is fix for #4459

### CODE REVIEW

File: clients/lsp-angular.el [hunk 1]
```diff
@@ -75,13 +75,13 @@ Has no effects when `lsp-clients-angular-language-server-command' is set."
...
-                "--ngProbeLocations"
-                node-modules-path
+                "ngserver"
+                "--stdio"
                 "--tsProbeLocations"
                 node-modules-path
-                "--stdio"))
+                "--ngProbeLocations"
+                (f-join node-modules-path "@angular/language-server/node_modules/")
+                ))
```
Comment (jcs090218) [inline clients/lsp-angular.el:84]: Can you avoid having trailing `))`? Thanks!

Comment (XenHunt) [reply clients/lsp-angular.el]: Well, If I understand correctly, I changed trailing `))`

Comment (jcs090218) [reply clients/lsp-angular.el]: I meant put it on the same line:

```diff
+ (f-join node-modules-path "@angular/language-server/node_modules/")))
- (f-join node-modules-path "@angular/language-server/node_modules/")
-                 ))
```

Comment (XenHunt) [reply clients/lsp-angular.el]: My bad, now it is applied

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Thank you for your contribution! 🚀

**Pull request merged**